unittests/src/main/java/uk/ac/manchester/tornado/unittests/virtualization/TestsVirtualLayer.java

```diff
@@ -18,24 +18,21 @@
 
 package uk.ac.manchester.tornado.unittests.virtualization;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNotNull;
-
 import java.util.Arrays;
 import java.util.stream.IntStream;
 
-import org.junit.Ignore;
-import org.junit.Test;
+import org.junit.*;
 
 import uk.ac.manchester.tornado.api.TaskSchedule;
 import uk.ac.manchester.tornado.api.TornadoDriver;
 import uk.ac.manchester.tornado.api.TornadoRuntimeCI;
 import uk.ac.manchester.tornado.api.annotations.Parallel;
 import uk.ac.manchester.tornado.api.runtime.TornadoRuntime;
-import uk.ac.manchester.tornado.unittests.common.TornadoTestBase;
+import uk.ac.manchester.tornado.unittests.tools.Exceptions.UnsupportedConfigurationException;
+
+import static org.junit.Assert.*;
 
-public class TestsVirtualLayer extends TornadoTestBase {
+public class TestsVirtualLayer {
```

Comment:
Let's keep this class as an extension of `TornadoTestBase`. We have a clean-up between tests.

Reply:
Done!